src/components/DocumentationTopic.vue

```diff
@@ -426,6 +429,15 @@ export default {
       || (downloadNotAvailableSummary && downloadNotAvailableSummary.length)
       || (primaryContentSections && primaryContentSections.length)
     ),
+    viewMoreLink: ({
+      interfaceLanguage,
+      objcPath,
+      swiftPath,
+      normalizePath,
+    }) => (
+      interfaceLanguage === Language.objectiveC.key.api
```

Comment:
I think we also want to include the objc query parameter in the URL for a link to an Objective-C destination.

It will technically work without it but only because of logic we have for preserving the language choice which would add extra computation that isn't needed when we already know we want to go to the objc variant.